frontend/src/services/routes/routes.ts

```diff
@@ -24,7 +24,12 @@ export const routes = {
 
             return `/weights${queryString !== "" ? `?${queryString}` : ""}`
         },
-        single: (slug: string) => `/weights/${slug}`
+        single: (slug: string, options?: { tab: string }) => {
+            if (!options || options.tab === "overview")
+                return `/weights/${slug}`
+
+            return `/weights/${slug}?tab=${options.tab}`
```

Comment:
Hier könnten wir den Aufbau ändern. 

Du hast es so gemacht:

- Checken wir ob es keine options gibt 
- => wenn ja return default 
- => wenn nein options mit tab

Wenn man eine weitere Option jetzt hinzufügt geht das nicht mehr so einfach. Daher eher wie oben machen:

mit URLSearchParams query objekt bauen und das in den default geben